docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -89,6 +89,70 @@ Timeout period (in milliseconds) for downstream requests. Default is 5000.
 --downstream-http-request-timeout=3000
 ```
 
+### downstream-http-tls-enabled
+
+Enable or disable [TLS for server connections](../../Concepts/TLS.md).
+
+```bash tab="Syntax"
+--downstream-http-tls-enabled
```

Comment:
Needs the `[=<true|false>]` I assume?  ie, how to you disable this?

Reply:
What is the default?

Reply:
Done.